src/main/java/core/basesyntax/dao/impl/MessageDaoImpl.java

```diff
@@ -13,20 +13,20 @@ public MessageDaoImpl(SessionFactory sessionFactory) {
     }
 
     @Override
-    public Message create(Message entity) {
+    public Message create(Message message) {
         Session session = null;
         Transaction transaction = null;
         try {
             session = factory.openSession();
             transaction = session.beginTransaction();
-            session.persist(entity);
+            session.persist(message);
```

Comment:
what about this https://github.com/mate-academy/hibernate-cascade/pull/122#discussion_r644623018?